### PR TITLE
PSMDB-1325 Release notes 6.0.9-7

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   docs/release_notes/6.0.9-7.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: variables.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PBM Variables set for HTML output
 # See also mkdocs.yml plugins.with-pdf.cover_subtitle and output_path
 
-release: '6.0.8-6'
+release: '6.0.9-7'
 version: '6.0'
-release_date: 2023-08-08
+release_date: 2023-09-11
```
Comment (igorsol) [inline variables.yml]: There is diffrent release date in index.md

**Review by janwieremjewicz — APPROVED** [review]

File: docs/release_notes/6.0.9-7.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Percona Server for MongoDB 6.0.9-7 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.9 Community Edition.
+
+It is based on [MongoDB 6.0.9 Community edition](https://www.mongodb.com/docs/v6.0/release-notes/6.0/#6.0.9---aug-14--2023) and supports the upstream protocols and drivers.
+
+
+## Release Highlights
+
+* The ability to [configure AWS STS endpoint](../aws-iam-setup.md#configure-aws-sts-endpoint) improves authentication and connectivity with AWS services.
```
Comment (igorsol) [inline docs/release_notes/6.0.9-7.md]: I thought this ability was added in previous release (with PSMDB-1291).

**Review by igorsol — CHANGES_REQUESTED** [review]

File: docs/release_notes/6.0.9-7.md (hunk 2 above)
Comment (nastena1606) [reply docs/release_notes/6.0.9-7.md]: Updated, thanks for the catch!

File: variables.yml (hunk 1 above)
Comment (sandraromanchenko) [inline variables.yml]:
```suggestion
release_date: 2023-09-13
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Review by igorsol — APPROVED** [review]

**Pull request merged**